src/ts/model/widget_instance.ts

```diff
@@ -25,9 +25,9 @@ export class WidgetInstance {
     currentWorkspace.create<WidgetInstance>(this);
 
     // this.WidgetCallbackClass = instance_loader.getInstance<any>(window, "Widget" + widget.alias, this.id);
-    this.WidgetCallbackClass = new window["Widget" + widget.alias](this.id); 
+    this.WidgetCallbackClass = new window["Widget" + widget.alias](this.id);
 
-    frontend.insertWidgetInstance(this, this.WidgetCallbackClass.clbkCreated);
+    frontend.insertWidgetInstance(this, () => this.WidgetCallbackClass.clbkCreated());
```

Comment:
Great, it solves some problems I had in the past. We don't need the "self" attribute as workaround doing like this. =)